digdag-tests/src/test/java/acceptance/td/TdResultExportIT.java

```diff
@@ -0,0 +1,148 @@
+import static org.hamcrest.Matchers.isEmptyOrNullString;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assume.assumeThat;
+
+public class TdResultExportIT
+{
+    private TDClient client;
+    private String database;
+    private String table;
+    private String savedQuery;
+    private String jobId;
+    private String connectionId;
+    private static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
+    private OkHttpClient httpClient;
+
+    @Before
+    public void setUp()
+            throws Exception
+    {
+        assumeThat(TD_API_KEY, not(isEmptyOrNullString()));
+        assumeThat(TD_API_ENDPOINT, not(isEmptyOrNullString()));
+
+        client = TDClient.newBuilder(false)
+                .setApiKey(TD_API_KEY)
+                .setEndpoint(TD_API_ENDPOINT)
+                .build();
+        database = "tmp_" + UUID.randomUUID().toString().replace('-', '_');
+        client.createDatabase(database);
+        String queryName = "test_" + UUID.randomUUID().toString().replace('-', '_');
+        createQuery(queryName);
+        TDSavedQueryStartRequest req = TDSavedQueryStartRequest.builder()
+                .name(queryName)
+                .scheduledTime(new Date())
+                .build();
+        this.jobId = client.startSavedQuery(req);
+        this.table = "test_" + UUID.randomUUID().toString().replace('-', '_');
+        client.createTableIfNotExists(database, table);
+        this.httpClient = new OkHttpClient();
+    }
+
+    @Test
+    public void testSubmitResultExportJob()
+            throws IOException
+    {
+        String testConnectorName =  "test_" + UUID.randomUUID().toString().replace('-', '_');
+        String json = "{\"description\":null,\"name\":\"" + testConnectorName + "\"," +
+                "\"settings\":{\"api_key\":\"\",\"api_hostname\":\"\"}," +
+                "\"shared\":false,\"type\":\"treasure_data\"}";
+
+        RequestBody body = RequestBody.create(JSON, json);
+        Request request = new Request.Builder()
+                .url("https://" + TD_API_ENDPOINT + "/v4/connections")
+                .header("authorization", "TD1 " + TD_API_KEY)
+                .post(body)
+                .build();
+        Response response = httpClient.newCall(request).execute();
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode jsonNode = objectMapper.readTree(response.body().string());
+        connectionId = jsonNode.get("id").asText();
+
+        String resultConnectionSettings = "{\"user_database_name\":\"" + this.database + "\"," +
+                "\"user_table_name\":\"" + this.table + "\"}";
+
+        TDExportResultJobRequest req = TDExportResultJobRequest.builder()
+                .jobId(this.jobId)
+                .resultConnectionId(connectionId)
+                .resultConnectionSettings(resultConnectionSettings)
+                .build();
+        client.submitResultExportJob(req);
+    }
+
+    @After
```

Comment:
There are multiple After methods.
Can we expect the execution order of these method?
Or don't you need to expect the order of call ?
Please explain.

Reply:
Here is outdated version, and latest version has only one `@After`.
https://github.com/treasure-data/digdag/pull/1240/files#diff-e1bf7d5b9435fc3a3e179ce39369575dR192